wpilibc/src/main/native/cpp/commands/InstantCommand.cpp

```diff
@@ -11,7 +11,37 @@ using namespace frc;
 
 InstantCommand::InstantCommand(const wpi::Twine& name) : Command(name) {}
 
+InstantCommand::InstantCommand(Subsystem& subsystem) : Command(subsystem) {}
+
 InstantCommand::InstantCommand(const wpi::Twine& name, Subsystem& subsystem)
     : Command(name, subsystem) {}
 
+InstantCommand::InstantCommand(std::function<void()> func)
+    : InstantCommand() {
```

Comment:
No need to call the default constructor `InstantCommand()`. It's called automatically.